calendar.go

```diff
@@ -275,26 +275,35 @@ type Calendar struct {
 }
 
 func NewCalendar() *Calendar {
+	return NewCalendarFor("Arran Ubels")
```

Comment:
I completely forgot about this. -- That shouldn't be there. Perhaps make it arran4?